src/i18n/pl_PL.ts

```diff
@@ -0,0 +1,1047 @@
+        },
+        resetWarningSetting: {
+            title: 'Resetuj ustawienia',
+            text:
+                'Czy na pewno chcesz zresetować to ustawienie<b>{setting}</b>modułu<b>{module}</b>do jego wartości domyślnej?',
+            close: 'Anuluj',
+            reset: 'Reset',
+        },
+        closeWarning: {
+            title: 'Niezapisane zmiany',
+            text:
+                'Dokonałeś zmian w ustawieniach, które nie zostały jeszcze zapisane. Zresetuj je, odrzuć lub zapisz, aby zamknąć ustawienia.',
+            close: 'Zamknij',
+        },
+        changeList: {
+            true: 'On',
+            false: 'Off',
+        },
+    },
+} as { [moduleId: string]: { [key: string]: unknown } };
+moduleRootFiles
+    .keys()
+    .forEach(key => (modules[key.split('/')[2]] = moduleRootFiles(key)));
+
+const t = {} as { [key: string]: unknown };
+
+// Commented as dir ./en_US does not exist currently
+// furtherFiles
+//     .keys()
+//     .forEach(
+//         key => (t[key.split('/')[1].replace(/\..*$/, '')] = furtherFiles(key))
+//     );
+
+export default {
+    modules,
+    ...t,
+    error: {
+        title: 'LSS Manager: Błąd',
+        msg:
+            'Jeśli ten błąd występuje często, zgłoś go zespołowi LSSM!',
+    },
+    warnings: {
+        version: {
+            title: 'Zła wersja LSS Manager',
+            text:
+                'Drogi użytkowniku, niestety odkryliśmy, że nie masz najnowszej wersji LSS Manager. Najnowsza wersja to {version}, ale ty masz {curver}. Proszę przeładować grę  (Ctrl + F5, na urządzeniach Apple polecenie + R), to powinno naprawić błąd. Jeśli błąd będzie się powtarzał, zgłoś go zespołowi! W przypadku użycia złej wersji nie możemy zagwarantować pełnej funkcjonalności LSS-Manager.',
+            close: 'Zamknij wiadomość i przeładuj grę (zalecane)',
+            abort: 'Zmaknij wiadomość bez przeładowania',
+        },
+    },
+    globalSettings: {
+        name: 'Ustawienia główne',
+        labelInMenu: {
+            title: 'Etykieta zamiast ikony w menu',
+            description:
+                'Wyświetla prostą etykietę na pasku nawigacji zamiast logo LSSM',
+        },
+    },
+    vehicles: {
+       0: {
+            caption: 'Ciężki samochód gaśniczy',
+            color: '#990000',
+            coins: 25,
+            credits: 5_000,
+            minPersonnel: 2,
+            maxPersonnel: 2,
+            wtank: 5_000,
+        },
+       1: {
+            caption: 'Średni samochód gaśniczy',
+            color: '#990000',
+            coins: 25,
+            credits: 5_000,
+            minPersonnel: 2,
+            maxPersonnel: 2,
+            wtank: 2_500,
+        },
+       2: {
+            caption: 'SD',
+            color: '#990000',
+            coins: 30,
+            credits: 10_000,
+            minPersonnel: 1,
+            maxPersonnel: 1,
+            wtank: 0,
+            special: 'Wymagane po zbudowaniu 3 remiz,

```

Comment:
please add a `'` before the `,` in this line 😉

Reply:
btw same thing at the other `special` lines (130, 140, 159, 171, 211, 221, 241, 253, 265288, 320, 330, 340, 370, 382, 392)

Reply:
I have a question: where can I find the building ID (even from version 3)?

Reply:
![afbeelding](https://user-images.githubusercontent.com/65117490/100207145-87874d80-2f07-11eb-967e-344a89d07c61.png)
 The only buildings I don't know are the Alliance buildings.

Reply:
0: "Jednostka Ratowniczo-Gaśnicza"
1: "Szkoła Pożarnicza"
2: "Stacja Pogotowia Ratunkowego"
3: "Akademia Służb Ratownictwa Medycznego"
4: "Szpital"
5: "Stacja helikopterów"
6: "Komenda Policji"
7: "Centrum Powiadamiania Ratunkowego"
8: "Akademia Policyjna"
13: "Stacja Helikopterów Policji"
14: "Miejsce koncentracji Sił i Środków"
15: "Stacja WOPR"
18: "Remiza"
19: "Posterunek Policji"
20: "Podstacja Pogotowia Ratunkowego"

Reply:
ok, thx